src/main.rs

```diff
@@ -175,6 +175,7 @@ fn main() {
         println!("{count} diagnostics issued.");
 
         if matches.is_present("print_grammar") {
+            db.grammar_pass();
```

Comment:
That's the better fix. But there is probably still a typo ("depdending") somewhere in the code ?

Reply:
Indeed, fixed in last commit!

Reply:
Typo fixed, so let's resolve this.